commands.py

```diff
@@ -141,6 +179,33 @@ def run(self, session: Session, _: sublime.Edit) -> None:
         )
 
 
+class CopilotGetPanelCompletionsCommand(CopilotTextCommand):
+    @_provide_session()
+    def run(self, session: Session, _: sublime.Edit) -> None:
+        params = prepare_completion_request(view=self.view)
+        if not params:
+            return
+
+        panel_id = "copilot://{}".format(self.view.id())
+        params["panelId"] = panel_id
+
+        set_copilot_view_setting(self.view, "panel_id", panel_id)
```

Comment:
If we introduce `ViewPanelCompletionManager`, all this could be part of it

Reply:
Done in [d641cff](https://github.com/TheSecEng/LSP-copilot/pull/18/commits/d641cff22ca9270ea70a9b2de1d34c2ed077b7b4)